include/stdio.h

```diff
@@ -107,11 +106,19 @@ typedef long long fpos_t;
 #define	SEEK_END	2
 #define	SEEK_SET	0
 #define	TMP_MAX		64		/* very hard to set correctly */
-#define	stderr	(&_IO_stream[2])
-#define	stdin	(&_IO_stream[0])
-#define	stdout	(&_IO_stream[1])
+//#define	stderr	(&_IO_stream[2])
+//#define	stdin	(&_IO_stream[0])
+//#define	stdout	(&_IO_stream[1])
 #define	_IO_CHMASK	0377		/* mask for 8 bit characters */
 
+extern FILE *const stdin;
+extern FILE *const stdout;
+extern FILE *const stderr;
+
+#define stdin  (stdin)
```

Comment:
POSIX requirement, inherited from musl. std* files must be macros.

The C standards C89/C99 (the last one I even looked up right now) say 
those identifiers are macros - according to stdio.h from glibc, which 
does the following:

   /* Standard streams.  */
   extern struct _IO_FILE *stdin;          /* Standard input stream.  */
[...]
   /* C89/C99 say they're macros.  Make them happy.  */
   #define stdin stdin
[...]